tests/Dom_2/dom.js

```diff
@@ -0,0 +1,51 @@
+const file_path = process.env.FILE_PATH;
+const webdriver = require('selenium-webdriver');
+const assert = require('chai').assert;
+const { By } = require("selenium-webdriver");
+
+describe("Testing Scholars Project", function () {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder()
+        .forBrowser('chrome')
+        .build();
+
+    var actions = driver.actions();
+
+    before(function setupWebdriver(done) {
```

Comment:
I see that there is `done` variable been used. How is it been passed to function?